### PR TITLE
Reject unsupported column types during AID labeling.

### LINKED ISSUE #411: Reject unsupported AID column types during labeling.
At the moment, we only support `integer`, `bigint` and `text/varchar` column types as AIDs.
We should check if the type is supported when the column is labeled, and issue an error right then if it isn't, instead of relying on the query execution error, which comes in a completely different context.

### PR DESCRIPTION
**Pull request by cristianberneanu**
Closes #411.

### CODE REVIEW

File: test/expected/misc.out [hunk 1]
```diff
@@ -197,3 +197,6 @@ EXPLAIN SELECT city FROM test_customers ORDER BY 1;
                ->  Seq Scan on test_customers
 (6 rows)
 
+-- Reject unsupported column types during AID labeling
+SECURITY LABEL FOR pg_diffix ON COLUMN test_customers.discount IS 'aid';
```
Comment (pdobacz) [inline test/expected/misc.out:201]: wouldn't this test be more tight, if the attempt was to set a column with a supported type? Also, the comment above is not in sync with the assertion.

Comment (cristianberneanu) [reply test/expected/misc.out:201]: Specifying a supported type is the happy path, which is tested everywhere else. I wanted to test here specifically that unsupported types are rejected.
I don't understand your last comment. It looks to me that they are in sync.

Comment (pdobacz) [reply test/expected/misc.out:201]: "unsupported column types" vs "requires direct access level"? the case in `admin.out` looks good but this seems off.

Comment (pdobacz) [reply test/expected/misc.out:201]: or maybe this is just a leftover, and you only want the `admin.out` one?

Comment (cristianberneanu) [reply test/expected/misc.out:201]: Ah, ok, I get it now :). You're right, I committed some wrong stuff by mistake. Thanks for catching it!

Comment (pdobacz) [reply test/expected/misc.out]: heh, it looked like a quite legit test, hence my confusion :)

**Review by pdobacz — APPROVED** [review]

**Pull request merged**